src/deploy-cromwell-on-azure/deploy-cromwell-on-azure.csproj

```diff
@@ -68,6 +87,33 @@
     <EmbeddedResource Include="scripts\env-04-settings.txt" />
     <EmbeddedResource Include="scripts\env-12-local-my-sql-db.txt" />
     <EmbeddedResource Include="scripts\env-13-postgre-sql-db.txt" />
+    <EmbeddedResource Include="scripts\helm\Chart.yaml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Since all these are embedded into the deployer's assembly, why are they also copied to the output directory?

Reply:
I need to write the final values.yaml file with the rest of the chart before it deployed. What should the build action be for these None?

Reply:
If you're rewriting it during deployment, keep it as `EmbeddedResource`, and remove `CopyToOutputDirectory`. Things like this in the output directory are only of possible use if the user is expected to manually interact with it without using the deployer (either before or after).

Reply:
Some users will need to interact with it though, if they are doing manually helm deployment.

Reply:
At what point and how do they interact with it? Do they start with the non-personalized version coming from git, or do they start with one personalized by the deployer?

Reply:
All the files are non-personalized except for values.yaml, the Helm program takes the directory as input to deploy the application to AKS. All the yaml files represent the CoA deployment, values.yaml can't be written until the deployer has the resource names, environment variables etc.